meson.build

```diff
@@ -111,6 +111,12 @@ wlr_deps += [
 	math,
 ]
 
+if host_machine.system().startswith('freebsd')
```

Comment:
It doesn't matter a lot, but isn't it possible to check for equality here?

Reply:
I copied this from sway's meson.build, and didn't consider if `startswith` really is necessary here. Checked it now, it works with `==`, commiited the change.